<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FSSE</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="bEst">
      <property name="text">
       <string>&amp;Estad√≠sticas</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bSave">
      <property name="text">
       <string>&amp;Guardar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bQuit">
      <property name="text">
       <string>&amp;Salir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bQuit</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
